packages/extension-base/src/background/handlers/State.ts

```diff
@@ -271,7 +284,7 @@ export default class State {
     };
   };
 
-  public udateCurrentTabsUrl (urls: string[]) {
+  public updateCurrentTabsUrl (urls: string[]) {
```

Comment:
Just as a side note: When we release this - this should be mentioned as a BREAKING CHANGE - it's quite minor but nonetheless will be nice to have in the CHANGELOG and release message. I do like this though.

Reply:
should preserve the original method and just mark it as deprecated and add the new one with the fixed name

Reply:
I've reintroduced `udateCurrentTabsUrl` and marked as deprecated.